docker-compose.yml

```diff
@@ -13,5 +13,40 @@ include:
   # Selenium services for UI testing
   - path: ./docker/developers/docker-compose.selenium.yml
 
+  # MinIO S3 Emulation
+  - path: ./docker/developers/docker-compose.minio.yml
+
   # The dashboard server
   - path: ./docker/developers/docker-compose.dashboard.yml
+
+services:
+  all-services:
+    image: alpine
```

Comment:
Curious if we've considered `ubuntu-server` versus `alpine` for consistency?

Reply:
No no. This just prints out some text to the screen. A libc+busybox distro is all that is necessary.